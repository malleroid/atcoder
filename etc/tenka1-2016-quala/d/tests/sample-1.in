10
A B
A C
A D
B E
B F
B G
C H
F I
I J
