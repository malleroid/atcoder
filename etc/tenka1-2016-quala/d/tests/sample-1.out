8 12
............
...D...E....
...|...|....
.C-A---B---F
.|.....|...|
.|.....G.J-I
.H..........
............
